releases/release-7.1.6.md

```diff
@@ -0,0 +1,300 @@
+---
+title: TiDB 7.1.6 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 7.1.6.
+---
+
+# TiDB 7.1.6 Release Notes
+
+Release date: November 21, 2024
+
+TiDB version: 7.1.6
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v7.1/production-deployment-using-tiup)
+
+## Compatibility changes
+
+- Set a default limit of 2048 for DDL historical tasks retrieved through the [TiDB HTTP API](https://github.com/pingcap/tidb/blob/release-7.5/docs/tidb_http_api.md) to prevent OOM issues caused by excessive historical tasks [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```

Comment:
```suggestion
- Set a default limit of 2048 for DDL historical tasks retrieved through the [TiDB HTTP API](https://github.com/pingcap/tidb/blob/release-7.1/docs/tidb_http_api.md) to prevent OOM issues caused by excessive historical tasks [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```